packages/instantsearch.js/src/lib/routers/history.ts

```diff
@@ -207,11 +207,25 @@ class BrowserHistory<TRouteState> implements Router<TRouteState> {
    * See: https://github.com/algolia/instantsearch.js/issues/790
    */
   public createURL(routeState: TRouteState): string {
-    return this._createURL({
+    const url = this._createURL({
       qsModule: qs,
       routeState,
       location: this.getLocation(),
     });
+
+    try {
```

Comment:
I could extract this into a function so that it's only executed in dev, or wrap it in `if(__DEV__)`, not sure what's best :/